.gitlab-ci.yml

```diff
@@ -75,6 +76,14 @@ workflow:
     - if: $CI_PIPELINE_SOURCE == "schedule"
     - if: $CI_COMMIT_REF_NAME =~ /^v[0-9]+\.[0-9]+.*$/              # i.e. v1.0, v2.1rc1
 
+### stage:                         fmt
+
+fmt:
+  stage:                           fmt
+  <<:                              *docker-env
+  script:
+    - cargo fmt --verbose --all -- --check
```

Comment:
Substrate itself actually uses `nightly` https://github.com/paritytech/substrate/blob/f00ec46a5fd4136d48b91a5f9122f2c1f41f6ad0/.gitlab-ci.yml#L248

It could even be that some of the settings are nightly only too, not sure.